h/views/accounts.py

```diff
@@ -137,6 +138,8 @@ def _js_config(self) -> dict[str, Any]:
             },
         }
 
+        inject_login_urls(self.request, js_config)
```

Comment:
This injects the URLs for the three **Continue with `<PROVIDER>`** links into the `js-config` for the `/login` page.